lib/material/internal/material_tree.flow

```diff
@@ -469,7 +473,7 @@ MTree2T(manager : MaterialManager, parent : MFocusGroup, m : MTree<?>, m2t : (Ma
 			extractStruct(m.style, MTreeExpanded(make(makeSet()))).expanded,
 			!contains(m.style, MTreeDisableExpanded()),
 			extractMColor(parent, m.style, MPrimaryColor()),
-			isSome(multiSelection),
+			fOr(const(isSome(multiSelection)), fand(const(listSelectionMultiple), listSelectionEnabledB)),
```

Comment:
fOr(const(isSome(multiSelection)), eitherMap(listSelection, \ls -> fand(const(ls.multiple), ls.enabled), const(false)))